lib/cluster.js

```diff
@@ -142,7 +142,11 @@ module.exports = function (oss) {
       yield that._checkAvailable(true);
```

Comment:
I think we should add a options to ignore `_checkAvailable` instead change the error logic.